docs/maui/converters/multi-math-expression-converter.md

```diff
@@ -0,0 +1,125 @@
+---
+title: MultiMathExpressionConverter - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: "The MultiMathExpressionConverter is a converter for multiple math expressions."
+ms.date: 03/16/2022
+---
+
+# MultiMathExpressionConverter
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `MultiMathExpressionConverter` is a converter for multiple math expressions.
+
+The `Convert` calculates the incoming expression string with multiple variables and returns a `double` result.
+
+## Syntax
+
+### XAML
+
+The `MultiMathExpressionConverter` can be used as follows in XAML:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="CommunityToolkit.Maui.Sample.Pages.Converters.MultiMathExpressionConverterPage">
+
+    <ContentPage.Resources>
+        <ResourceDictionary>
+            <toolkit:MultiMathExpressionConverter x:Key="MultiMathExpressionConverter" />
+        </ResourceDictionary>
+    </ContentPage.Resources>
+
+    <Label HorizontalOptions="Center" >
+        <Label.FormattedText>
+            <FormattedString>
+                <Span Text="x0 + x1 + x2 = "/>
+                <Span>
+                    <Span.Text>
+                        <MultiBinding Converter="{StaticResource MultiMathExpressionConverter}" ConverterParameter="x0 + x1 + x2">
```

Comment:
As a reader of the documentation I would like to know some general rules around what should be provided in the `ConverterParameter` and how it applies to the `values` that are bound. For example if I supplied a 4th binding value would it have any impact? If I don't supply `x1` would it have an impact? And is there any correlation between the value and the name in the expression or could I in theory write `? + ? + ?` in the expression and it still work?

This might be better placed at the top where we explain what the converter is.